lesson3/matrix_utils/sum_of_diagonals_or_diagonal_of_matrix.py

```diff
@@ -0,0 +1,40 @@
+from is_matrix_square import square_checker
+
+
+def sum_of_diagonals_of_matrix(matrix):
+    if square_checker(matrix):
+        sum_of_diagonals = 0
+        length = len(matrix)
+        for i in range(length):  # sum of diagonal
+            sum_of_diagonals += matrix[i][i]
+
+        matrix.reverse()
```

Comment:
Bad approach. You don't need another matrix.

Reply:
Done